### ISSUE TITLE
Install directory should be removed!

### ISSUE THREAD

**Issue by Danielyan86** [bitnami/bitnami-docker-testlink#81]
where is the Install directory in the container?

**Comment by carrodher** [comment]
Thanks for letting us know, if you have any other questions, please do not hesitate to create a new issue.

**Comment by margru** [comment]
Hi, I believe this issue should be reopened. The "install" directory should be removed automatically, or at least e.g. when any envvar is enabled, so that the user doesn't need to take care of this every time the container is started... It's actually a security threat.

**Comment by rafariossaa** [comment]
Hi,
  Which container version are you using ? 
  I used the latest one, `1.9.20`, and I could not find a install directory inside testlink:
```
I have no name!@fc3e23d9d1c9:/$ ls  /opt/bitnami/testlink/
BUYING_SUPPORT.TXT                                  firstLogin.php
CHANGELOG                                           gui
CODE_REUSE                                          index.php
Dockerfile                                          lib
LICENSE                                             licenses
README.md                                           linkto.php
README.plugins                                      lnl.php
cfg                                                 locale
config.inc.php                                      login.php
config_db.inc.php                                   logout.php
custom                                              logs
custom_config.inc.php                               lostPassword.php
custom_config.inc.php.example                       ltx.php
custom_config.inc.php.example.github_oauth          microsoftoauth.php
custom_config.inc.php.github.testlinkOauthProvider  plugin.php
custom_config.inc.php.google_oauth                  plugins
docs                                                third_party
error.php                                           upload_area
extra                                               vendor
```

**Comment by margru** [comment]
Hmm, OK, maybe it's fixed in 1.9.20. I am using 1.9.19 as the newest one has several issues preventing me from using it... OK then, good to know.

**Comment by rafariossaa** [comment]
Hi,
  As we release images with the lastest stable version, we are not releasing more images for 1.9.19.

**Comment by margru** [comment]
> Hi,
> As we release images with the lastest stable version, we are not releasing more images for 1.9.19.

It's a pitty as 1.9.20 is unusable and 1.9.19 (and maybe the older ones as well) is not secure...

**Comment by carrodher** [comment]
I am sorry but we follow the same approach for releases that the upstream project follows. If the upstream project supports several branches at the same time (like PHP, MariaDB, PostgreSQL, etc) we provide the different active branches until they reach EOL, but if the upstream project doesn't support different branches and they implement a rolling releases approach, we follow the same method.

In this case, once released 1.9.20 there are no new releases for 1.9.19, see https://github.com/TestLinkOpenSourceTRMS/testlink-code/blob/testlink_1_9_20_fixed/CHANGELOG.

Can you describe what makes 1.9.20 unusable? In this way, we can work on fixing any issues with that image

**Comment by margru** [comment]
I think it might be a problem with Testlink 1.9.20 itself, not with the image. But, actually, I cannot be sure of course. 

For example, after switching to a different language in the GUI, I started getting just error messages in the GUI and I even couldn't change the language back. The only easy way was to re-deploy it. Next, some of the links didn't work as expected. However, I didn't have that much time to experiment with that, so once I saw it's buggy I switched to 1.9.19 which works well for me - except the install directory.
